meta-balena-common/recipes-bsp/u-boot/patches/env_resin.h

```diff
@@ -47,7 +47,7 @@
        "resin_flasher_skip=0 \0" \
        \
        "resin_find_root_part_uuid=" \
-               "part uuid ${resin_dev_type} ${resin_dev_index}:${resin_root_part} resin_root_part_uuid\0" \
+               "fsuuid ${resin_dev_type} ${resin_dev_index}:${resin_root_part} resin_root_part_uuid\0" \
```

Comment:
We'll need to enforce enabling `CONFIG_CMD_FS_UUID` in the uboot configs. There is some partial work already in meta-balena for this. It isn't ideal though. uboot configs are messy and split into defconfigs and header configs.

Reply:
thanks @ZubairLK could you point me to the partial work you mention? This is something that will probably need to happen on the same release so I'd like to get it moving.

Reply:
See https://meet.google.com/linkredirect?authuser=0&dest=https%3A%2F%2Fgithub.com%2Fbalena-os%2Fmeta-balena%2Fpull%2F1581%2Fcommits%2F7051a1bda83d53e08148eea953ee2653fd915743